pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: layout.pug
+navigationTitle: Management cluster application requirements
+title: Management cluster application requirements
+menuWeight: 40
+excerpt: Management cluster application minimum resources and storage requirements
+beta: false
+enterprise: true
+---
+
+This section details requirements for management cluster-specific applications. For the list of all available platform applications, see [Platform Application Configuration Requirements](../../../projects/applications/platform-applications/application-requirements/).
+
+This table describes the workspace platform applications specific to the management cluster, minimum resource requirements, and minimum persistent storage requirements.
```

Comment:
Do we we want this page to be the "internal" apps that are ONLY deployed on the mgmt cluster? The other platform apps are workspace-scoped and we can link there instead of duplicating them in this table here (as is linked in the above paragraph).

The list of internal apps that we would then just include on this page are:
- centralized-grafana (enterprise)
- centralized-kubecost (enterprise)
- chartmuseum
- dkp-insights-management
- gitea
- karma (enterprise)
- kubefed
- thanos (enterprise)
- traefik-forward-auth-mgmt

Reply:
@gracedo @cneth - do we want to run with Grace's suggestion above? We need to get this PR closed by end of day, please!

Reply:
@Colin1841  - I am happy with  Grace's suggestion....

Reply:
Table updated, per @gracedo recommendation.

Reply:
@cneth @gracedo - can you take a quick look at the Jira ticket and this thread from John Miller: https://mesosphere.slack.com/archives/C01LNF6KML5/p1649248000737439
Do we need Loki in this table, or is this a moot point now with the smaller version of the table?

Reply:
loki would be included in the workspace platform apps table, so no need to duplicate it here. this table should be the internal apps that are ONLY deployed on the mgmt cluster